content/docs/studio/get-started.md

```diff
@@ -42,13 +42,21 @@ details]).
 
    </admon>
 
-3. Iterative Studio parses your project's `dvc.yaml` file to identify data,
-   metrics, plots and hyperparameters.
-
-   If you are not using DVC, you can separately
-   [indicate which files contain metrics and hyperparameters](/doc/studio/user-guide/projects-and-experiments/configure-a-project#custom-metrics-and-parameters).
-   However, we strongly recommend using DVC to avail of all the features of
-   Iterative Studio.
+3. Iterative Studio identifies experiments and their data, metrics, plots and
+   hyperparameters form the following sources:
+
+   1. **`dvc.yaml` file**: Details of experiments saved as Git commits are
+      fetched from this file.
+   2. **Git refs**: Details of experiments pushed using `dvc exp push` are
+      fetched from here.
+   3. **Metrics and plots logged by [DVCLive]**: To [track live metrics and
+      plots][live-metrics-and-plots] of running experiments, you should set the
+      `DVC_STUDIO_TOKEN` environment variable and use [DVCLive] in your training
```

Comment:
@mvshmakov Can you pls confirm that the var is called `DVC_STUDIO_TOKEN` (not `STUDIO_ACCESS_TOKEN`)?

Reply:
[From the code](https://github.com/iterative/dvclive/blob/0ec68604ddd7ed7f1c0f64fb4e40048bad37fed3/src/dvclive/live.py#L143), it seems like we have two options: `DVC_STUDIO_TOKEN` and `STUDIO_TOKEN`.

I personally was always using `STUDIO_TOKEN`, but I can see that it may be not specific enough and theoretically clash with another tool.

@daavoo could you please chime in and explain why we want to have two ways of providing the token + which one we should recommend in the docs?

Reply:
> could you please chime in and explain why we want to have two ways of providing the token

What you mentioned (`I can see that it may be not specific enough and theoretically clash with another tool.`) was the main motivation for adding the `DVC_` prefix. 
We support both for backward compatibility.

> which one we should recommend in the docs?

Please try to recommend `DVC_STUDIO_TOKEN` from now on.

Reply:
Thanks @mvshmakov @daavoo